### PR TITLE
Update PKGBUILD

### PR DESCRIPTION
**Pull request by nikolowry**
Add prepare function to set GOPATH and install vendor dependencies. Also change install permissions from 644 to 755.

### CODE REVIEW

**Comment by gchaincl** [comment]
why would you go get dependencies if they are vendored?

File: contrib/archlinux/PKGBUILD [hunk 1]
```diff
@@ -14,12 +14,18 @@ install=
 source=("$pkgname"::'git+https://github.com/gchaincl/httplab.git')
 md5sums=(SKIP)
 
+prepare() {
+	cd "$pkgname"
+	GOPATH="$srcdir"
```
Comment (gchaincl) [inline contrib/archlinux/PKGBUILD]: Not sure if this is going to work as expected, the directory structre will be $srcdir/httplab, in order to make it work, we should have $srcdir/github.com/gchaincl/httplab.
I'm I missing something?

File: contrib/archlinux/PKGBUILD [hunk 2]
```diff
@@ -14,12 +14,18 @@ install=
...
+
 build() {
 	cd "$pkgname"
 	go build
 }
 
 package() {
 	cd "$pkgname"
-	install -D -s -m644 httplab "${pkgdir}/usr/bin/httplab"
+	install -Dsm755 httplab "${pkgdir}/usr/bin/httplab"
```
Comment (gchaincl) [inline contrib/archlinux/PKGBUILD]: 👍

Comment (gchaincl) [inline contrib/archlinux/PKGBUILD]: which go version are you using?

Comment (nikolowry) [inline contrib/archlinux/PKGBUILD]: I only have the community version of go installed - which is at 1.8.  Not a golang dev and the only other package I use that has dependecies on go is git-lfs (which is what I used as a reference for GOPATH to get httplab to install locally with makepkg)

Attaching screenshot of the failed build if I try to install from AUR:
2017-03-01-144414_1336x1638_scrot

Comment (gchaincl) [inline contrib/archlinux/PKGBUILD]: Thanks for this, I'll try to reproduce the error and merge your fix

**Pull request closed without merging**